waspc/waspls/src/Wasp/LSP/DynamicHandlers.hs

```diff
@@ -119,3 +118,61 @@ sourceFilesChangedHandler msg = do
           logM $ "[watchSourceFilesHandler] Updating missing diagnostics for " ++ show fileUris
           publishDiagnostics uri
         Nothing -> pure ()
+
+-- | Register file watcher watcher for Prisma schema file.
+registerPrismaSchemaFileWatcher :: ServerM ()
+registerPrismaSchemaFileWatcher = do
+  -- We have two options here:
+  -- 1) Watch any schema.prisma file in the project directory tree
+  -- 2) Watch only the schema.prisma file in workspace root which might not be correct
+  --   if the user openned their IDE some other folder other than the workspace root.
+  let prismaSchemaGlob = "**/schema.prisma"
+
+  globPattern <-
+    -- NOTE: We use the workspace root, instead of the wasp root here, because
+    -- 1) The wasp root may not be known yet.
+    -- 2) Using the workspace root results only in the potential to consider
+    --    files that do not matter. Important files won't be missed (the workspace
+    --    root will necessarily contain the wasp root).
+    LSP.getRootPath >>= \case
+      Nothing -> do
+        logM "Could not access projectRootDir when setting up source file watcher. Watching any schema.prisma file instead of limiting to src/."
+        return prismaSchemaGlob
+      Just projectRootDir -> do
+        return $
+          if "/" `isSuffixOf` projectRootDir
+            then projectRootDir <> prismaSchemaGlob
+            else projectRootDir <> "/" <> prismaSchemaGlob
```

Comment:
Oh and also important, `</>` will use the right separator based on the system (Win vs Posix), while with `/` you assumed Posix so it won't work on Windows.

Reply:
But this also might be the probelm actually -> so the question is, here in this world of vscode, are all these paths always Posix because vscode guarantees that, or do they adhere to the underlying system (win vs posix(mac, linux))? You will need to know that to be able to correctly implement this.

Reply:
I'll try using the `</>` 👍 

It seems from the LS specification, it will always be a valid URI (posix): https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#uri

Reply:
`FP.</>` works great for this use-case 👍